src/style.css

```diff
@@ -0,0 +1,133 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, serif;
```

Comment:
reudundant, you don't inherit it anyways
```suggestion
```

Reply:
done